src/Symfony/Component/Messenger/Tests/Transport/RedisExt/RedisExtIntegrationTest.php

```diff
@@ -104,4 +102,77 @@ public function testConnectionSendDelayedMessagesWithSameContent()
         $this->assertEquals($body, $encoded['body']);
         $this->assertEquals($headers, $encoded['headers']);
     }
+
+    public function testConnectionBelowRedeliverTimeout()
+    {
+        // lower redeliver timeout and claim interval
+        $connection = Connection::fromDsn(getenv('MESSENGER_REDIS_DSN'), [], $this->redis);
+
+        $connection->cleanup();
+        $connection->setup();
+
+        $body = '{"message": "Hi"}';
+        $headers = ['type' => DummyMessage::class];
+
+        // Add two messages
+        $connection->add($body, $headers);
+
+        // Read first message with other consumer
+        $this->redis->xreadgroup(
+            'symfony',
+            'other-consumer2',
+            ['messages' => '>'],
```

Comment:
same as above get the stream name from `$connection->getStream()` as the stream could be different when testing locally (which was the case for me and I did wonder what it was failing 🙈 )